README.md

```diff
@@ -113,88 +117,47 @@ In organisaties waarin de gebruikers van veg2hab geen volledige local admin rech
 
 ### Gebruik in ArcGIS Pro
 
+#### Sequentiële omzetstappen
+
 De omzetting van vegetatiekarteringen naar habitattypekaarten gebeurt via de Python Toolbox `veg2hab.pyt`. De gehele omzetting verloopt via een aantal sequentiële stappen: 
 
 <img src="https://github.com/Spheer-ai/veg2hab/raw/master/images/toolbox_components.png" alt="new notebook" width="400"/>
 
-**Omschrijving van de stappen:**
-- `1a_digitale_standaard`: Laadt een vegetatiekartering in die de landelijke digitale standaard gebruikt. Deze bestaat uit een shapefile gecombineerd met een access database. Voor de shapefile kan de gebruiker verwijzen naar een bestandslocatie óf naar een kaart die reeds ingeladen is in ArcGIS Pro.
-- `1b_vector_bestand`: Laadt een vegetatiekartering in die alle benodigde informatie in de shapefile zelf heeft staan. Omdat shapefiles geen standaard kolomnamen hebben, dient de gebruiker hier een handvol inputvelden in te vullen, bijvoorbeeld welke kolom de te gebruiken landelijke typologie bevat. Voor de shapefile kan de gebruiker verwijzen naar een bestandslocatie óf naar een kaart die reeds ingeladen is in ArcGIS Pro.
+Omschrijving van de omzetstappen:
+- `1a_digitale_standaard`: Laadt een vegetatiekartering in die de landelijke digitale standaard gebruikt. Deze bestaat uit een vectorbestand (zoals een shapefile of geopackage) gecombineerd met een access database. Voor de shapefile kan de gebruiker verwijzen naar een bestandslocatie óf naar een kaart die reeds ingeladen is in ArcGIS Pro.
```

Comment:
Ik denk dat het goed is om hierbij ook de argumenten te beschrijven of niet?

Reply:
done